packages/stream-text/package.json

```diff
@@ -32,13 +32,14 @@
   "scripts": {
     "build": "pkgroll",
     "build:watch": "pkgroll --watch",
-    "test": "vitest run",
+    "test": "vitest run --coverage",
     "test:watch": "vitest"
   },
   "dependencies": {
     "@xsai/shared-chat": "workspace:"
   },
   "devDependencies": {
+    "@vitest/coverage-istanbul": "3.0.4",
```

Comment:
maybe `@vitest/coverage-v8`?

Reply:
Also, I think this dependency should be installed into workspace.

Reply:
> Also, this dependency should be installed in the workspace.

will other subprojects also use coverage? I'm not sure if it's reasonable to elevate this to the root.

Reply:
> maybe `@vitest/coverage-v8`?

according to vitest
> Vitest supports Native code coverage via [v8](https://v8.dev/blog/javascript-code-coverage) and instrumented code coverage via [istanbul](https://istanbul.js.org/).

use istanbul will be more appropriate?